nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/WebServer.java

```diff
@@ -147,6 +147,7 @@ class Builder implements io.helidon.common.Builder<Builder, WebServer>, Router.R
         private final HelidonServiceLoader.Builder<ServerConnectionProvider> connectionProviders =
                 HelidonServiceLoader.builder(ServiceLoader.load(ServerConnectionProvider.class));
 
+        private Config providersConfig;
```

Comment:
This should by default be `providersConfig = Config.empty()` to make sure that it is not null

Reply:
fixed